java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -211,19 +211,9 @@ class DatabaseImpl(
 
   /* Initializes the [PrimitiveType] values. */
   private fun initializePrimitiveTypes(db: SQLiteDatabase) {
-    // Populate the 'types' table with the primitive types. The id of the enum will be
-    // the Type ID used in the database.
-    val content = ContentValues().apply {
-      put("is_primitive", true)
-    }
-    PrimitiveType.values().forEach {
-      content.apply {
-        put("id", it.id)
-        put("name", it.name)
-      }
-      db.insertOrThrow(TABLE_TYPES, null, content)
-    }
-
+    // Populate the 'types' table with the sentinel value for reference types.
```

Comment:
A better comment would be something like this:
// We used to keep primitive types in the table, and used the sentinel to distinguish between primitive and entity types.
// Now we only keep entity types here, but the sentinel is kept for backward compatibility.